xbmc/cores/VideoPlayer/Edl.cpp

```diff
@@ -840,6 +840,50 @@ bool CEdl::InCut(const int iSeek, Cut *pCut) const
   return false;
 }
 
+bool CEdl::GetNearestCut(bool bPlus, const int iSeek, Cut *pCut) const
+{
+  if (bPlus) {
+    // Searching forwards
+    for (int i = 0; i < (int)m_vecCuts.size(); i++) 
```

Comment:
But you could use iterators instead of indices. They exist for both directions (`begin()` and `rbegin()` etc.)